mmtf-api/src/main/java/org/rcsb/mmtf/api/DataApiInterface.java

```diff
@@ -16,7 +16,7 @@
 public interface DataApiInterface {
 
 	/**
-	 * Returns a list of length N atoms of the X coords of the atoms as Integers.
+	 * Returns an array of length N atoms of the X coordinates of the atoms as integers.
 	 * They must be divided by 1000.0 to be in float form.
```

Comment:
We should point to a constant somewhere instead of writing 1000.0, then refer to the constant with {@value class#CONSTANT}